content/en/synthetics/api_tests/http_tests.md

```diff
@@ -71,6 +71,14 @@ After choosing the type of test you want to create ([`HTTP`][4], [`SSL`][5], [`T
   * **Proxy URL**: Specify the URL of the proxy the HTTP request should go through (`http://<YOUR_USER>:<YOUR_PWD>@<YOUR_IP>:<YOUR_PORT>`).
   * **Proxy Header**: Add headers to include in the HTTP request to the proxy.
 
+  {{% /tab %}}
+  
+  {{% tab "Privacy" %}}
+
+  * **Do not save response body**: Select this option to prevent response body from being saved at runtime. This can be helpful to ensure no sensitive data gets featured in your test results. It however needs to be used mindfully as it can make failure troubleshooting way more challenging. You can read more about our security recommendations [here][1].
```

Comment:
```suggestion
  * **Do not save response body**: Select this option to prevent response body from being saved at runtime. This can be helpful to ensure no sensitive data gets featured in your test results. Use mindfully as it can make failures troubleshooting more difficult. Read more about security recommendations [here][1].
```